EntAppProject/src/main/resources/templates/JBLogin.html

```diff
@@ -1,45 +1,102 @@
 <head>
-<title>Welcome to JobBook</title>
-<link rel="stylesheet" href="JobBookCSS.css">
+	<title>JobBook - Login</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+<!--===============================================================================================-->	
+	<link rel="icon" type="image/png" href="../static/images/icons/favicon.ico"/>
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/vendor/bootstrap/css/bootstrap.min.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/fonts/font-awesome-4.7.0/css/font-awesome.min.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/fonts/Linearicons-Free-v1.0.0/icon-font.min.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/vendor/animate/animate.css">
+<!--===============================================================================================-->	
+	<link rel="stylesheet" type="text/css" href="../static/vendor/css-hamburgers/hamburgers.min.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/vendor/animsition/css/animsition.min.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/vendor/select2/select2.min.css">
+<!--===============================================================================================-->	
+	<link rel="stylesheet" type="text/css" href="../static/vendor/daterangepicker/daterangepicker.css">
+<!--===============================================================================================-->
+	<link rel="stylesheet" type="text/css" href="../static/css/JobBookLoginUtil.css">
+	<link rel="stylesheet" type="text/css" href="../static/css/JobBookLogin.css">
+<!--===============================================================================================-->
 </head>
 <body>
- 
-<h1><span>Job</span>Book</h1>
- 
-<h2>Welcome to JobBook</h2>
-<p></p>
- 
-<form action="action_page.php" method="post">
-<div class="container">
-        <label for="username"><b>Username</b></label>
-        <input type="text" placeholder="Username:" name="username" required>
-        <br>
-        <br>
-        <label for="userpassword"><b>Password</b></label>
-        <input type="password" placeholder="Password" name="userpassword" required>
-        <br>
-        <form method="get" action="/JBHomeScreen">
-        <button type="submit">Login</button>

```

Comment:
I noticed that this page had the same form nesting issue that I noticed in the previous code review. Again, I am not sure this is possible and should be solved by having one form that pushes to a single endpoint that redirects to the home page after a successful login attempt.